gitbook-docs/market/user-guides/proof-producer.md

```diff
@@ -1,82 +1,88 @@
-# Proof Producer
+# Proof producer
 
 {% hint style="info" %}
-Ensure you have done the [Authentication](sign-up.md) setup before progressing
+Make sure you've performed the [authentication setup](sign-up.md) before continuing.
 {% endhint %}
 
-## Submit Ask order
+## Submit a proposal
 
-The Proof producers can submit asks for circuits and specify an accompanying cost.
+Proof producers can submit proposals for statements and specify an accompanying cost like this:
 
-```
-python3 scripts/ask_tools.py push --cost <cost of the ask> --key <key of the statement> 
+```bash
+python3 scripts/proposal_tools.py push --cost <cost of the proposal> --key <key of the statement> 
 ```
 
-## Order Status/Fetch Inputs
+## Check order status/fetch inputs
 
-The proof producer can check their ask with
+Producers can check their proposal with the following:
 
-```
-python3 scripts/ask_tools.py get --key <key of the ask> 
+```bash
+python3 scripts/proposal_tools.py get --key <key of the proposal> 
 ```
 
-Ask's status 'processing' means that the ask was matched with a bid. Now it is time to generate a proof for the proof producer.
+If the proposal's status is `processing`, it means that the proposal was matched with the request.
+Now it's time for the proof producer to generate the proof.
 
-First of all, the proof producer needs circuit definition:
-
-```
+First of all, the proof producer needs statement definition:
 
+```bash
 python3 scripts/statement_tools.py get --key <key of the statement> -o <output file> 
-
 ```
 
-Next, public input of the bid:
+Next, public input of the request:
 
-```
-python3 scripts/public_input_get.py --key <bid key> -o <output file path> 
+```bash
+python3 scripts/public_input_get.py --key <request key> -o <output file path> 
 ```
 
-## Proof Generation
+## Generate a proof 
 
-Execute the below to generate proof:
-
-```
+To generate proof execute the following:
 
+```bash
 cd build
-./bin/proof-generator/proof-generator --proof_out=<output file> --circuit_input=<statement from Proof Market> --public_input=<public input from Proof Market>
+   ./bin/proof-generator/proof-generator --proof_out=<output file> \
+   --circuit_input=<statement from Proof Market> --public_input=<public input from Proof Market>
```

Comment:
first line shouldn't be indented because it's the beginning of a command. Wrapped lines should be indented by 4 spaces.

Parameter values can be passed in two ways:
```
--param=value
--param value
```

Both are valid, but let's use the second option for consistency.
```suggestion
./bin/proof-generator/proof-generator \
    --proof_out <output file> \
    --circuit_input <statement from Proof Market>
    --public_input <public input from Proof Market>
```

Reply:
For these guides - fixed